### ISSUE TITLE
Don't exit the shell on syntax error

### ISSUE THREAD

**Issue by certik** [prefix-dev/shell#33]
```
C:\Program Files\Git$ cd ~
Error: Failed to execute

Caused by:
     --> 1:4
      |
    1 | cd ~
      |    ^---
      |
      = expected EOI, AND_IF, OR_IF, Stdout, StdoutStderr, cmd_suffix, or separator_op
error: process didn't exit successfully: `target\debug\shell.exe` (exit code: 1)
```
This exits the shell. Instead show the next prompt.

**Comment by prsabahrami** [comment]
This has been fixed in #38 
Support for miette should be added later. Switching from bug to enhancement.

**Comment by certik** [comment]
This issue is fixed. I opened a new one (https://github.com/prefix-dev/shell/issues/77) for miette.
